src/components/BlacklistedTagsSelector/index.tsx

```diff
@@ -0,0 +1,438 @@
+  value: MultiValue<SingleVal> | null;
+  onChange: (value: MultiValue<SingleVal> | null) => void;
+  components?: Partial<typeof components>;
+};
+
+const ControlledKeywordSelector = ({
+  defaultValue,
+  onChange,
+  components,
+  value,
+}: BaseSelectorMultiProps) => {
+  const intl = useIntl();
+
+  useEffect(() => {
+    const loadDefaultKeywords = async (): Promise<void> => {
+      if (!defaultValue) {
+        return;
+      }
+
+      const keywords = await Promise.all(
+        defaultValue.split(',').map(async (keywordId) => {
+          const res = await fetch(`/api/v1/keyword/${keywordId}`);
+          if (!res.ok) {
+            throw new Error('Network response was not ok');
+          }
+          const keyword: Keyword = await res.json();
+
+          return keyword;
+        })
+      );
+
+      onChange(
+        keywords.map((keyword) => ({
+          label: keyword.name,
+          value: keyword.id,
+        }))
+      );
+    };
+
+    loadDefaultKeywords();
+  }, [defaultValue, onChange]);
+
+  const loadKeywordOptions = async (inputValue: string) => {
+    const res = await fetch(
+      `/api/v1/search/keyword?query=${encodeURIExtraParams(inputValue)}`
+    );
+    if (!res.ok) {
+      throw new Error('Network response was not ok');
+    }
+    const results: TmdbKeywordSearchResponse = await res.json();
+
+    return results.results.map((result) => ({
+      label: result.name,
+      value: result.id,
+    }));
+  };
+
+  return (
+    <AsyncSelect
+      key={`keyword-select-blacklistedTags`}
+      inputId="data"
+      isMulti
+      className="react-select-container"
+      classNamePrefix="react-select"
+      noOptionsMessage={({ inputValue }) =>
+        inputValue === ''
+          ? intl.formatMessage(messages.starttyping)
+          : intl.formatMessage(messages.nooptions)
+      }
+      value={value}
+      loadOptions={loadKeywordOptions}
+      placeholder={intl.formatMessage(messages.searchKeywords)}
+      onChange={onChange}
+      components={components}
+    />
+  );
+};
+
+type BlacklistedTagsCopyButtonProps = {
+  value: string;
+};
+
+const BlacklistedTagsCopyButton = ({
+  value,
+}: BlacklistedTagsCopyButtonProps) => {
+  const intl = useIntl();
+  const [isCopied, setCopied] = useClipboard(value, {
+    successDuration: 1000,
+  });
+  const { addToast } = useToasts();
+
+  useEffect(() => {
+    if (isCopied) {
+      addToast(intl.formatMessage(messages.copyBlacklistedTags), {
+        appearance: 'info',
+        autoDismiss: true,
+      });
+    }
+  }, [isCopied, addToast, intl]);
+
+  return (
+    <Tooltip
+      content={intl.formatMessage(messages.copyBlacklistedTagsTip)}
+      tooltipConfig={{ followCursor: false }}
+    >
+      <button
+        onClick={(e) => {
+          e.preventDefault();
+          setCopied();
+        }}
+        className="input-action"
+        type="button"
+      >
+        <ClipboardDocumentIcon />
+      </button>
+    </Tooltip>
+  );
+};
```

Comment:
Why don't you use the already existing CopyButton instead of duplicating it here?

Reply:
I was messing around with the button initially and forgot to merge my changes back into CopyButton. Fixed!